src/sherpa_ai/tools.py

```diff
@@ -77,12 +78,21 @@ def _arun(self, query: str) -> str:
 
 class SearchTool(BaseTool):
     name = "Search"
+    config: AgentConfig = AgentConfig()
     description = (
         "Access the internet to search for the information. Only use this tool when "
         "you cannot find the information using internal search."
     )
 
+    def _add_gsite(self, query: str) -> str:
+        # check if the gsite is none
+        if self.config.gsite:
```

Comment:
a more concise way of writing this:

`return query + " " + self.config.gsite if self.config.gsite else query`

the comment on 88 is not needed

Reply:
The problem is addressed! Thanks for your suggestion